 * QA Notice: Package has poor programming practices which may compile
 *            fine but exhibit random runtime failures.
 * xX.c:101: warning: implicit declaration of function 'exit'
 * tcpdump.c:93: warning: implicit declaration of function 'exit'
 * signal_handler.c:91: warning: implicit declaration of function 'exit'
 * sleep.c:65: warning: implicit declaration of function 'exit'
 * signal_handler.c:91: warning: implicit declaration of function 'exit'
 * sleep.c:65: warning: implicit declaration of function 'exit'
 * sleep.c:65: warning: implicit declaration of function 'exit'

=== modified file 'src/common/err.h'
--- src/common/err.h	2009-04-03 11:02:19 +0000
+++ src/common/err.h	2009-04-03 11:06:00 +0000
@@ -50,6 +50,8 @@
 extern int debug;
 #endif
 
+#include <stdlib.h>
+
 /*
  * We define five functions for reporting errors, warnings and debug messages:
  * err()   - Fatal error.  Pass exit code followed by static string

